Makefile

```diff
@@ -3,6 +3,8 @@ SHELL=/usr/bin/env bash
 
 .PHONY: test
 test:
+	cat ./plugin.yaml | grep uri: | cut -c 10-200 > plugin_artifacts_urls.txt
+	if wget -i ./plugin_artifacts_urls.txt --server-response --quiet "${url}" 2>&1 | grep -q "404"; then echo 'Artifacts links are broken'; exit 1;else echo 'Artifacts links are valid';fi  
```

Comment:
Doesnt this mean that when i release i will always fail the test (until the release will finish)?

Reply:
that's true, this code makes sure that if you try to change the artifacts link and they are still not available, it will fail.